### PR TITLE
Fix SQL error in purge expired data cron

### PR DESCRIPTION
**Pull request by ausi**
Executing this cron runs into the following error: `You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ':tstamp' at line 1`

`Connection::prepare()` does not support named parameters AFAIK (or maybe that changed in a recent Doctrine version?).

### CODE REVIEW

**Comment by ausi** [comment]
Not sure if this needs fixing in Contao 4.9 too? as it might be Doctrine 3.0 related?

**Comment by fritzmg** [comment]
Seems odd, why would they drop support for that? 🤔

File: core-bundle/src/Cron/PurgeExpiredDataCron.php [hunk 1]
```diff
@@ -48,7 +49,10 @@ private function cleanTable(string $table, int $period): void
             return;
         }
 
-        $stmt = $this->connection->prepare(sprintf('DELETE FROM %s WHERE tstamp<:tstamp', $table));
-        $stmt->executeStatement(['tstamp' => time() - $period]);
```
Comment (leofeyer) [inline core-bundle/src/Cron/PurgeExpiredDataCron.php:52]: Isn‘t the actual problem that the array key has to be `':tstamp'` instead of `'tstamp'`?

Comment (Toflar) [reply core-bundle/src/Cron/PurgeExpiredDataCron.php:52]: No, this changed in DBAL 3.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**